nodejs/eks/cluster.ts

```diff
@@ -85,10 +85,11 @@ export interface CoreData {
     nodeSecurityGroupTags?: InputTags;
 }
 
-function createOrGetInstanceProfile(name: string, parent: pulumi.ComponentResource, instanceRoleName?: pulumi.Input<aws.iam.Role>, instanceProfileName?: pulumi.Input<string>): aws.iam.InstanceProfile {
+export function createOrGetInstanceProfile(name: string, parent: pulumi.ComponentResource, instanceRoleName?: pulumi.Input<aws.iam.Role>, instanceProfileName?: pulumi.Input<string>): aws.iam.InstanceProfile {
```

Comment:
I don’t think we want to export this, it’s just an internal helper - right?

Reply:
maybe not.. which is why I did not do that at first. But it did not show up in the generated js file - I guess its optimized away.. which was kinda annoying. Your call.

Reply:
removed export